.github/workflows/deploy.yml

```diff
@@ -0,0 +1,78 @@
+name: Deploy
+on:
+  workflow_run:
+    workflows: [ Check code ]
+    types: [ completed ]
+    branches: [ main ]
```

Comment:
This configuration is not checked in thid PR becauce `Note: This event will only trigger a workflow run if the workflow file is on the default branch.`